src/kernels/dynamic_igemm/igemm_v4r1_wrw_dynamic.s

```diff
@@ -1474,7 +1474,7 @@ L_debug_code_seg:
     v_sub_u32 v[v_sst_a_os], v[v_sst_a_os], v[v_tmp+1]
     v_lshrrev_b32 v[v_sst_a_os], 2, v[v_sst_a_os]
 
-    global_store_dword v[v_end:v_end+1], v[v_a], s[s_tmp+12:s_tmp+13]
+    ;global_store_dword v[v_end:v_end+1], v[v_a], s[s_tmp+12:s_tmp+13]
```

Comment:
removed? no need store to global here?

Reply:
No, this is a debug branch, and it will not be ran unless developer want to debug the asm code. As we will refactor the asm kernels by GTC, I removed this piece of code.

Reply:
Yes. I think debug purpose should not exist in our production.

Reply:
I agree. I will remove this branch.